django_fido/views.py

```diff
@@ -49,11 +53,13 @@ class Fido2ViewMixin(object):
                        see https://www.w3.org/TR/webauthn/#enumdef-attestationconveyancepreference
     @cvar attestation_types: Allowed attestation format types.
         If None, all attestation formats except `none` are allowed.
+    @cvar user_verification: The desired USER_VERIFICATION level.
```

Comment:
I'd use `Requirements of user verification` to match the terms used in specs.

Reply:
I copied that from the fido2 package but the specs are certainly better.

Reply:
Sadly, the `fido2` descriptions are not very descriptive.